<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #rgb(99, 128, 143)</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>191</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #435761    ;
border: 0px solid black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewFiles">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>View Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFiles">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #becbd2  ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Add Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryChanges">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Query Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryPast">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Query the Past</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:helvetica;
font-size:30px;
color: #435761  ;</string>
   </property>
   <property name="text">
    <string>Add new version</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Select File Group</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: #7d97a5 ;
font-size:14px;
</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="groupName">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Group Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="nameField">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>180</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(125, 151, 165);
alternate-background-color: rgb(125, 151, 165);
font-size: 20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Upload File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="uploadField">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(125, 151, 165);
alternate-background-color: rgb(125, 151, 165);
font-size: 15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>330</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Version Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(125, 151, 165);
alternate-background-color: rgb(125, 151, 165);
font-size: 15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Version Number</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="versionNum">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>350</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(125, 151, 165);
alternate-background-color: rgb(125, 151, 165);
font-size: 15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>500</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>Add Version</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>Date </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionSuccess">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>510</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;</string>
   </property>
   <property name="text">
    <string>Version added successfully! </string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>300</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)
</string>
   </property>
   <property name="text">
    <string>Please select a file!</string>
   </property>
  </widget>
  <widget class="QLabel" name="entername1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>380</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)
</string>
   </property>
   <property name="text">
    <string>Enter a name!</string>
   </property>
  </widget>
  <widget class="QLabel" name="entername2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)
</string>
   </property>
   <property name="text">
    <string>Enter a name!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="userField">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>430</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(125, 151, 165);
alternate-background-color: rgb(125, 151, 165);
font-size: 15px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #303e45 ;
  font-size: 22px;
font-family: helvetica;</string>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editFile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Edit File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="elemHistory">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #7d97a5 ;
 ;  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string> Element Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewOld">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #7d97a5 ;
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Access Past Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="monitor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">  border: none;
background-color: #7d97a5 ;

  color: white;
  padding: 15px 32px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
font-family: helvetica;
border: 1px solid white;

</string>
   </property>
   <property name="text">
    <string>Montior Files</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
